capstone/src/test/java/com/google/sps/servlets/announcements/ScheduleAnnouncementServletTest.java

```diff
@@ -93,6 +99,47 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  @Test
+  public void doGet_desiredBehavior() throws IOException {
+    helper.setEnvEmail(TEST_EMAIL).setEnvAuthDomain("gmail.com").setEnvIsLoggedIn(true);
+    when(request.getParameter(Constants.PROPERTY_NAME)).thenReturn(SAMPLE_CLUB_NAME);
+    when(request.getUserPrincipal()).thenReturn(principal);
+    when(principal.getName()).thenReturn(TEST_EMAIL);
+
+    Entity announcement1 = new Entity(Constants.FUTURE_ANNOUNCEMENT_PROP);
+    announcement1.setProperty(Constants.AUTHOR_PROP, TEST_EMAIL);
+    announcement1.setProperty(Constants.CLUB_PROP, SAMPLE_CLUB_NAME);
+    announcement1.setProperty(Constants.CONTENT_PROP, SAMPLE_NEW_CONTENT);
+    announcement1.setProperty(Constants.TIME_PROP, SAMPLE_TIME);
+    announcement1.setProperty(Constants.EDITED_PROP, false);
+    this.datastore.put(announcement1);
+
+    JsonArray response = getServletResponse(servlet);
+
+    int expectedSize = 1;
+    Assert.assertEquals(expectedSize, response.size());
```

Comment:
you only use expected size once, I would just put it as Assert.assertEquals(1, response.size()) in that case